src/Components/WebAssembly/Server/src/DebugProxyLauncher.cs

```diff
@@ -46,11 +46,12 @@ private static async Task<string> LaunchAndGetUrl(IServiceProvider serviceProvid
         var executablePath = LocateDebugProxyExecutable(environment);
         var muxerPath = DotNetMuxer.MuxerPathOrDefault();
         var ownerPid = Environment.ProcessId;
-
+        var noProxyEnvVar = Environment.GetEnvironmentVariable("NO_PROXY");
+        var ignoreProxyForLocalAddress = (noProxyEnvVar is not null && (noProxyEnvVar.Equals("localhost") || noProxyEnvVar.Equals("127.0.0.1"))) ? "--IgnoreProxyForLocalAddress True" : "";
```

Comment:
What if there is some other value assigned to the NO_PROXY variable?
The current logic simply ignores it, without any log or warning. Maybe at least worth logging a message about it so that customers who choose to use some custom value know what's going on from logs ?